src/lib/redux/inventoryDataSlice.ts

```diff
@@ -66,8 +89,21 @@ const inventoryDataSlice = createSlice({
     addInventory(state, action) {
       state.sitesInventory[action.payload.siteId].siteInventory.push(action.payload);
     },
+    addPurchaseRequest(state, action) {
+      const siteId = action.payload.siteId;
+      delete action.payload.siteId;
+      state.sitesInventory[siteId].purchaseRequests.push(action.payload);
+    },
+    // NOTE: name conflicts with request.js
```

Comment:
Not really sure how to handle this -- should I rename to avoid confusion?

Reply:
We're probably fine here especially if it's behind a `setter` function. If anything, we could probably use namespaces to deal with it?